lib/removeUnusedElements.js

```diff
@@ -0,0 +1,235 @@
+'use strict';
+var Cesium = require('cesium');
+var ForEach = require('./ForEach');
+var hasExtension = require('./hasExtension.js');
+
+var defined = Cesium.defined;
+
+module.exports = removeUnusedElements;
+
+/**
+ * Removes unused elements from gltf.
+ * This function currenlty only works for accessors, buffers, and bufferViews.
+ *
+ * @param {Object} gltf A javascript object containing a glTF asset.
+ * @returns {Object} The glTF asset with the added pipeline extras.
```

Comment:
Fix the description, looks like a copy-paste error.

Reply:
Done.